docs/docs/setup/auth/user/callback.md

```diff
@@ -0,0 +1,232 @@
+---
```

Comment:
I commented on this in another doc, but it happened multiple times here, too, so I'm not sure it's intended, and will just add a global note like this.

Should 

> shibauthorizer

be

> `shibautorizer`

?